### PR TITLE
[llvm] Increase the inline limit when using llvm in jit mode.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: mono/mini/method-to-ir.c [hunk 1]
```diff
@@ -4240,12 +4244,20 @@ mono_method_check_inlining (MonoCompile *cfg, MonoMethod *method)
...
-		} else
+		} else {
 			inline_limit = INLINE_LENGTH_LIMIT;
+			llvm_jit_inline_limit = LLVM_JIT_INLINE_LENGTH_LIMIT;
+		}
 		inline_limit_inited = TRUE;
 	}
-	if (header.code_size >= inline_limit && !(method->iflags & METHOD_IMPL_ATTRIBUTE_AGGRESSIVE_INLINING))
+
+	if (COMPILE_LLVM (cfg) && !cfg->compile_aot)
```
Comment (kumpera) [inline mono/mini/method-to-ir.c]: what does this mean, that we use the smaller limit when AOT'ing with LLVM?

Comment (vargaz) [reply mono/mini/method-to-ir.c]: We will use the same limit we have now.

**Comment by luhenry** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins backport 2018-04

**Pull request merged**